src/ngraph/attribute_adapter.cpp

```diff
@@ -337,6 +337,28 @@ namespace ngraph
         m_buffer_valid = false;
     }
 
+#ifdef __APPLE__
+    // size_t is not uint64_t on OSX
```

Comment:
The comment mentions uint64_t but the code references int64_t. Is the comment or the code wrong?
Same comment for the header file below.

Reply:
The comment is correct. It is trying to say that the existing adapter for uint64_t (size_t on other platforms) does not work on OS X. So a new adapter is needed. int64_t is in the implementation since get/set method for all integer type is based on int64_t.

Reply:
The extra code is not needed once we have #4666 merged. The extra code is a temporary solution to remove build failures on macOS.

Reply:
To clarify, on Macos, `size_t` is not one of the `uintx_t` types.